lib/kamal/cli/healthcheck.rb

```diff
@@ -3,6 +3,7 @@ class Kamal::Cli::Healthcheck < Kamal::Cli::Base
 
   desc "perform", "Health check current app version"
   def perform
+    raise "The primary host is not configured to run Traefik" unless KAMAL.config.role(KAMAL.config.primary_role).running_traefik?
```

Comment:
`KAMAL.config.role(KAMAL.config.primary_role)` might be too verbose.

In my original changes `KAMAL.config.primary_role` was an instance of `Role`.

However, with #577, `KAMAL.config.primary_web_role` was introduced as a String.

I didn't want to change that to make the review simpler. I can submit a PR afterwards with a proposed cleanup if you think it's interesting

Reply:
Hi @yoelcabo - yes if you could raise another PR to convert `KAMAL.config.primary_role` to a `Role` that would be great. I'll merge this PR now 🙏